linode/manager
# feat: [M3-7052] - Add NodeBalancers table under Firewall / Devices

## Description 📝
Added the Nodebalancer table and functionality under the Nodebalancer tab in Firewalls page.

## Major Changes 🔄
- Nodebalancers have their own dedicated table
- You can remove the Nodebalancers
- There is a toast notification when removing a Nodebalancer from a firewall

## Preview 📷
**Include a screenshot or screen recording of the change**

> **Note**: Use `<video src="" />` tag when including recordings in table

| Before  | After   |
| ------- | ------- |
| Content | Content |

## How to test 🧪
1. Navigate to http://localhost:3000/firewalls
2. Select any of the firewalls
3. Navigate to the Nodebalancer tab
4. Add multiple Nodebalancers to the firewall
5. Ensure the Nodebalancers get added, you can sort the Nodebalancers and you can remove the Nodebalancers
6. Ensure when removing the Nodebalancers, a toast notification pops up.
